lark/tree_templates.py

```diff
@@ -84,7 +84,9 @@ def __default__(self, data, children, meta):
 
         var = self._conf.test_var(tree)
         if var:
-            return self._vars[var]
+            value = self._vars.get(var)

```

Comment:
If the variable doesn't exist, shouldn't we throw an exception instead?

Reply:
Yeah, that makes sense. Since it wasn't documented I had to make a guess. I'll switch it to raise a sub-class of `LarkError`.